store/datastore.go

```diff
@@ -30,6 +30,10 @@ type DataStore interface {
 	//limits
 	GetLimit(ctx context.Context, name string) (*model.Limit, error)
 
+	//storage settings
+	GetStorageSettings(ctx context.Context, tenantID string) (*model.StorageSettings, error)
```

Comment:
I'm not sure how to do it better considering porting the changes to deployments-enterprise. In `deployments` service `tenantID string` is not used but will be used in `deployments-enterprise`.
@tranchitella How to do it better - have different interfaces for those 2 services or make it unified?

Reply:
keep tenantID here, and assume it's an empty string if multi-tenancy is not enabled